Expensify/App
# Add validation to requestCall page and add policy check

### Details

https://github.com/Expensify/App/issues/4861#issuecomment-906962896
### Fixed Issues

$ https://github.com/Expensify/App/issues/4861

### Tests | QA Steps

   1. Go to staging.new.expensify.com
   2. Login with any account
   3. Search for Concierge
   4. Click on the Call icon
   5. Don't write anything on the phone number and click call.
   6. Nothing should happen.
   7. Enter a phone number and click call.


### Tested On


### Screenshots


#### Web | Mobile Web | Desktop


![image](https://user-images.githubusercontent.com/24370807/132749404-543e2051-60e3-49c9-a75a-4a888d7463e1.png)
![image](https://user-images.githubusercontent.com/24370807/132765539-e85959e0-1487-4f95-b7d0-e3980f4c7175.png)


#### iOS


#### Android


![Screenshot 2021-09-10 00:52:18](https://user-images.githubusercontent.com/24370807/132749710-aae0e30e-e4ff-40b1-b012-0b6a7301768a.png)
